omnigibson/robots/tiago.py

```diff
@@ -550,16 +669,30 @@ def finger_joint_names(self):
 
     @property
     def usd_path(self):
-        return os.path.join(gm.ASSET_PATH, "models/tiago/tiago_dual_omnidirectional_stanford/tiago_dual_omnidirectional_stanford_33.usd")
+        return os.path.join(gm.ASSET_PATH, "models/tiago/tiago_dual_omnidirectional_stanford/tiago_dual_omnidirectional_stanford_33_with_wrist_cam.usd")
+        # return os.path.join(gm.ASSET_PATH, "models/tiago/tiago_dual_omnidirectional_stanford/tiago_dual_omnidirectional_stanford_33.usd")
+
+    @property
+    def simplified_mesh_usd_path(self):
```

Comment:
I know what this is used for, but I wonder if there's a way to make this more general (i.e.: have this property for all supported robots in OG)

Reply:
For now I'm just leaving a TODO.